javalib/src/main/scala/java/util/package.scala

```diff
@@ -14,6 +14,16 @@ package java
 
 package object util {
 
+  implicit private[util] class CompareNullablesOps(val self: Any) extends AnyVal {
+    @inline
+    def =:=(that: Any): Boolean = {
```

Comment:
I understand you modeled this after the `===` that was here before, but it's totally overkill for reference equality. Just use `a.asInstanceOf[AnyRef] eq b.asInstanceOf[AnyRef]` at call site. Especially since there seems to be only one call site.

Reply:
Sure.